ci-operator/step-registry/rosa/e2e/ocm-fvt/rosa-e2e-ocm-fvt-commands.sh

```
@@ -74,7 +77,17 @@ if [[ "${OCM_FVT_REPORT_JIRA:-true}" == "true" ]]; then
 echo "Running ocmtest: ${ocmtest_args[*]}"
+exit_code=0
 podman run \
   "${podman_args[@]}" \
   quay.io/redhat-services-prod/ocmci/ocmci:latest \
-  ocmtest "${ocmtest_args[@]}"
\ No newline at end of file
+  ocmtest "${ocmtest_args[@]}" || exit_code=$?
+
+# Copy JUnit XML to ARTIFACT_DIR with junit- prefix for Sippy ingestion
+find "${ocm_fvt_output}" -type f -name '*.xml' -print0 | while IFS= read -r -d '' xml_file; do
+  rel_path="${xml_file#${ocm_fvt_output}/}"
```

Shellcheck (0.11.0)

[info] Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

(SC2295)